src/About.jl

```diff
@@ -130,13 +130,15 @@ style of the output.
 module About
 
 using StyledStrings: @styled_str, Face, face!, addface!
-using JuliaSyntaxHighlighting: highlight
+using JuliaSyntaxHighlighting
 using InteractiveUtils
 
 @static if VERSION >=v"1.11-alpha"
     using Base: AnnotatedString, AnnotatedIOBuffer
+    global const highlight = identity
```

Comment:
```suggestion
    const highlight = identity
```
The `global` qualifier isn't necessary since this is `const` and you're already in global scope here.